internal/cli/atlas/deployments/podman/client.go

```diff
@@ -114,11 +174,14 @@ func (o *client) machineStart(ctx context.Context) error {
 	return nil
 }
 
-func (o *client) Ready(ctx context.Context) error {
+func Installed() error {
 	if _, err := exec.LookPath("podman"); err != nil {
-		return ErrPodmanNotReady
+		return ErrPodmanNotFound
 	}
+	return nil
+}
```

Comment:
this probably belong in the podman package

Reply:
I'm confused, it's under podman package now, no?

Reply:
oh sorry, I saw `internal/cli/atlas/deployments/` and that's cli logic, the popman package should be a top level package `internal/podman/client.go` it's not related to the commands

Reply:
ahh gotcha, let me try to refactor this quick and see if it's ok